csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/Usings.cs

```diff
@@ -1,2 +1 @@
 global using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Deprecation;
```

Comment:
You still need the global using here, but it should now be `Deprecation.StringUtils`. Also note that global usings require the word `global` in the front, so this line should be `global using Deprecation.StringUtils;`